scope

```diff
@@ -9,7 +9,14 @@ usage() {
     exit 1
 }
 
-SCOPE_IMAGE=weaveworks/scope
+SCRIPT_VERSION="(unreleased version)"
+if [ "$SCRIPT_VERSION" = "(unreleased version)" ] ; then
+    IMAGE_VERSION=latest
+else
+    IMAGE_VERSION=$SCRIPT_VERSION
+fi
```

Comment:
My bash-fu may be weak, but it looks like this will always take the first branch. Do you want something like—

```bash
IMAGE_VERSION=${SCRIPT_VERSION:="(unreleased version)"}
```

?